ibm/resource_ibm_is_image.go

```diff
@@ -0,0 +1,276 @@
+package ibm
+
+import (
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.ibm.com/Bluemix/riaas-go-client/clients/compute"
+	iserrors "github.ibm.com/Bluemix/riaas-go-client/errors"
+)
+
+const (
+	isImageHref            = "href"
+	isImageName            = "name"
+	isImageOperatingSystem = "operating_system"
+	isImageStatus          = "status"
+	isImageVisibility      = "visibility"
+
+	isImageProvisioning     = "provisioning"
+	isImageProvisioningDone = "done"
+	isImageDeleting         = "deleting"
+	isImageDeleted          = "done"
+)
+
+func resourceIBMISImage() *schema.Resource {
+	return &schema.Resource{
+		Create:   resourceIBMISImageCreate,
+		Read:     resourceIBMISImageRead,
+		Update:   resourceIBMISImageUpdate,
+		Delete:   resourceIBMISImageDelete,
+		Exists:   resourceIBMISImageExists,
+		Importer: &schema.ResourceImporter{},
+
```

Comment:
Add timeout block
```
Timeouts: &schema.ResourceTimeout{
			Create: schema.DefaultTimeout(10 * time.Minute),
			Delete: schema.DefaultTimeout(10 * time.Minute),
		},
```

with default value

Reply:
added the timeout block..